### PR TITLE
Simulator clarifications 

### PR DESCRIPTION
**Pull request by hamishwillee**
These address issues as per #357. Images not yet updated.

### CODE REVIEW

File: en/simulation/README.md [hunk 1]
```diff
@@ -48,23 +48,23 @@ Message | Direction | Description
 [HIL_RC_INPUTS_RAW](http://mavlink.org/messages/common#HIL_RC_INPUTS_RAW) | Sim to PX4 | The RAW values of the RC channels received.
 
 
-## Standard MAVLink UDP Ports
+## Default PX4 MAVLink UDP ports
```
Comment (hamishwillee) [inline en/simulation/README.md:51]: FMI, capitalise "Ports"

File: en/simulation/README.md [hunk 2]
```diff
@@ -48,23 +48,23 @@ Message | Direction | Description
...
 
-PX4 defines a number of standard UDP broadcast ports that ground control stations (e.g. *QGroundControl*, Offboard APIs and simulators) can listen to in order to connect to the autopilot:
+By default, PX4 uses well-defined MAVLink UDP ports for communication with ground control stations (e.g. *QGroundControl*), Offboard APIs (e.g. DroneCore, MAVROS) and simulator APIs (e.g. Gazebo). These ports are:
 
-- Offboard APIs: 14540
-- Ground Control Stations: 14550  (*QGroundControl* automatically connects to PX4 broadcasting on this port).
-- Simulators: 14560
+* Port **14540** is used for communication with offboard APIs. PX4 broadcasts data towards this port, and Offboard APIs are expected to listen.
```
Comment (hamishwillee) [inline en/simulation/README.md]: @cyberang3l  Might the first two be better as below (removing "broadcast" removes the "feeling" that this is single direction communication).

* Port **14540** is used for communication with offboard APIs. Offboard APIs are expected to listen for connections on this port. 
* Port **14550** is used for communication with ground control stations. GCS are expected to listen for connections on this port. *QGroundControl* listens to this port by default.

Comment (cyberang3l) [reply en/simulation/README.md]: I agree with this. Better remove `broadcast`.

**Comment by cyberang3l** [comment]
As you explained to me in #357, the ports are not `well-defined` by means of being `standardized`. However these ports are the *defacto* standard across different autopilot software solutions so I would call them `commonly established`. Thus, I would only change this part:

`By default, PX4 uses well-defined MAVLink UDP ports for communication`

to this:

`By default, PX4 uses commonly established MAVLink UDP ports for communication`

The word `commonly` redirects me to think about something that is, ...well..., common to use.
The word `establish` makes me think about something has managed to rightfully claim a status: to become common in this case.

Figures are coming.

**Comment by cyberang3l** [comment]
Black dashed lines: Internal component communication.
Purple solid lines: Serial communication.
Red dashed lines: MAVLink UDP communication. Can be wireless or not.

px4_simulator_messages

px4_sitl_overview

px4_hitl_overview

**Comment by hamishwillee** [comment]
@cyberang3l  Thanks very much! I have integrated your suggestions an the images and merged. 

Minor point is that in SITL/HITL diagram we refer to comms as MAVLink API Communication. MAVLink is actually a protocol, so this should probably be  just "MAVLink" or "MAVLink Protocol" or "MAVLink Communication". Thoughts?

**Comment by cyberang3l** [comment]
I agree. It should be a plain `MAVLink communication`.
I make a quick fix on that and send you the new figures.

**Comment by cyberang3l** [comment]
px4_hitl_overview

px4_sitl_overview

**Comment by hamishwillee** [comment]
Cheers, merged as above. Just as an aside, we could be *very* pedantic, that the HITL serial comms is actually MAVLink over serial. But we aren't going to be :-)

**Pull request merged**